CONTRIBUTING.md

```diff
@@ -225,8 +282,8 @@ func main() {
 }
 ```
 
-### VM Tests using QEMU
-For a package *foo*, put your unit tests inside `foo_test.go`. To test integration with other parts without mocking, write integration test in a file `integration_test.go`. In case you need to test against certain hardware, you can use a QEMU environment via `pkg/vmtest`. Put the setup and corresponding tests in `vm_test.go`
+### Integration Tests
+To test integration with other code/packages without mocking, write integration test in a file `integration_test.go`.
```

Comment:
Where does this `integration_test.go` belong? `/integration/generic-tests` or inside the package one is writing?

Reply:
If you want to test integration with other pkgs it belongs in the pkg you are writing.
`/integration/generic-tests` are for bigger scoped test regarding the complete u-root ecosystem / u-root command.